forum/Themes/Redsy/css/index.css

```diff
@@ -460,22 +460,83 @@ tr.catbg th a:link, tr.catbg th a:visited, tr.catbg2 td a:link, tr.catbg2 td a:v
 	color: #000;
 	background-color: #fff;
 }
+
 /* the page navigation area */
 .pagesection
 {
-	font-size: 0.9em;
 	padding: 0.2em;
 	overflow: hidden;
-	margin-bottom: 1px;
+	margin-top: 0.3em;
+	margin-bottom: 0.3em;
+}
+
+/* Main pagination links */
+.pagesection .pagelinks > *
```

Comment:
In general it's frowned upon to use wildcard selectors. How many different cases are you matching here?